modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -183,18 +191,24 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     return backupKeyShare;
   }
 
-  createUserKeychain(
-    userGpgKey: SerializedKeyPair<string>,
-    userKeyShare: KeyShare,
-    backupKeyShare: BackupKeyShare,
-    bitgoKeychain: Keychain,
-    passphrase: string,
-    originalPasscodeEncryptionCode?: string,
-    isThirdPartyBackup = false
-  ): Promise<Keychain> {
+  createUserKeychain({
+    userGpgKey,
+    userKeyShare,
+    backupKeyShare,
+    bitgoKeychain,
+    passphrase,
+    bitgoPublicGpgKey,
+    originalPasscodeEncryptionCode,
+    isThirdPartyBackup = false,
+  }: CreateEcdsaKeychainParams): Promise<Keychain> {
+    assert(bitgoKeychain);
```

Comment:
Will this assertion still be needed after making `bitgoKeychain` no optional?

Reply:
Nop not needed anymore after making it required. Good catch